internal/pkg/flink/internal/controller/input_controller.go

```diff
@@ -128,6 +128,10 @@ func (c *InputController) RunInteractiveInput() {
 			c.isSessionValid(err)
 			continue
 		}
+		// print status detail message if available
+		if processedStatement.StatusDetail != "" {
+			output.Printf("Status detail: %s\n", processedStatement.StatusDetail)
```

Comment:
Do we necessarily need to label this, or should just printing it be sufficient?
```suggestion
			output.Println(processedStatement.StatusDetail)
```

Reply:
I think just printing it could be sufficient. We might want to indicate that this is a response from the server, but is not required.